integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -137,10 +142,21 @@ def export(self, **kwargs):
 
             # Fetch course details from Courses API
             # pylint: disable=unsubscriptable-object
+            if course_details:
+                LOGGER.info(
+                    '[Integrated Channels] Currently exporting for course: {curr_course}. '
+                    'Course details already found for course: {found_course}'.format(
+                        curr_course=course_id,
+                        found_course=course_details['course_id'])
```

Comment:
I have a feeling this line will fail the same way as what we saw (perhaps there is a way to log this without extracting the field course_id ?